views/pages/login.ejs

```diff
@@ -20,7 +20,7 @@
                 </li>
                 <li>
                     <label for="password" id="password" name="password">password</label>
-                    <input type="text">
+                    <input type="password">
```

Comment:
The id and name attributes go on the input tag rather than the label.

Reply:
Sure, I will fix this